app/views/hiring_staff/vacancies/review.html.haml

```diff
@@ -197,9 +197,9 @@
         %dd.app-check-your-answers__answer
           - if @vacancy.application_link.present?
             = link_to @vacancy.application_link, class: 'govuk-link', 'aria-label': t('jobs.aria_labels.application_link_url', url: @vacancy.application_link) do
-              %span= t('jobs.application_link_url')
-              %br
               %span.govuk-body-s.govuk-link= @vacancy.application_link
+              %br
+              %span.govuk-body-s.govuk-link= t('jobs.application_link_url')
```

Comment:
Are these both intentionally small? Looks a bit odd to me. I would expect the URL to be normal size.

Reply:
Yes they are. In fact before the changes, the label "Link for more ... / (link opens ... )" was in a normal size and then the URL was smaller. Now we made it consistently small only.

Reply:
I guess it was because URLs can be too long sometimes? I did think of making all of them in normal size so I decided to stick to the original one. 

Let me try to see if I can get the context behind.

Reply:
I have restored this part to the original design (with the new descriptive text). Detailed reasons can be found in the trello card https://trello.com/c/Y1Qwnuy2/825-review-and-iterate-content-around-url-for-existing-application-page#comment-5cc043415f09db3da1ea4305